### DISCUSSION TITLE
Hovering Texts of Columns

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6589] [Help]
Hello All,

Before migrating to serenity 6.2.2 our site used to display column data when hover, but after migration it stopped.

This was before migration:
![image](https://user-images.githubusercontent.com/98172041/203116202-120b8136-7272-4474-92a8-bb17f5cc9dbe.png)

This is after migration:
![image](https://user-images.githubusercontent.com/98172041/203116500-a69616b8-5f96-4789-b6bf-e8ba34a0a7cb.png)

Is there anything to add to fix this?

**Comment by volkanceylan** [comment]
Hmm, it might be related to SleekGrid, e.g. our SlickGrid rewrite no longer using jquery for binding to events, and that plugin probably requires jquery events. We'll have a look at that.

**Comment by ibrahimGit123** [comment]
@VictorTomaili FYI we discussed this just now
